prereise/gather/griddata/hifld/data_process/demand.py

```diff
@@ -0,0 +1,81 @@
+import inspect
+import os
+
+import pandas as pd
+
+from prereise.gather.griddata import hifld
+from prereise.gather.griddata.hifld import const
+
+
+def assign_demand_to_buses(substations, branch, plant, bus):
+    """Using data on population by county and ZIP code, assign demand to substations,
```

Comment:
Maybe `demand` -> `demand parameter` to be specific?

Reply:
In the function name?

Reply:
Nope, in the doc strings.

Reply:
I made a tweak, let me know what you think.